apps/main-site/src/pages/browse.tsx

```diff
@@ -0,0 +1,21 @@
+import { BrowseCommunitiesPage } from '@answeroverflow/ui/src/components/pages/BrowseCommunitiesPage';
+import type { ServerPublic } from '~api/router/server/types';
+import { trpc } from '@answeroverflow/ui';
+
+export default function BrowseCommunitiesPageWeb({
+	servers,
+}: {
+	servers: ServerPublic[];
+}) {
+	return <BrowseCommunitiesPage servers={servers} />;
+}
+
+export function getServerSideProps() {
+	const servers = trpc.servers.getAllServers.useQuery().data;
```

Comment:
Servers need to be filtered to where kicked time != null

Reply:
Fixed